app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntryEffectHandler.kt

```diff
@@ -53,7 +53,32 @@ class BloodPressureEntryEffectHandler(
 ) {
   private val reportAnalyticsEvents = ReportAnalyticsEvents()
 
-  fun create(): ObservableTransformer<BloodPressureEntryEffect, BloodPressureEntryEvent> {
+  companion object {
```

Comment:
We should switch to this pattern for all effect handlers at some point for consistency.

Reply:
Will do!